src/Tizen.NUI/src/public/RenderEffects/RenderEffect.cs

```diff
@@ -33,6 +34,16 @@ namespace Tizen.NUI
     [EditorBrowsable(EditorBrowsableState.Never)]
     public class RenderEffect : BaseHandle
     {
+        /// <summary>
+        /// Enumeration for selecting how the mask source interprets pixel data
+        /// Alpha: Uses the alpha channel of the mask texture. (Default)
+        /// Luminance: Converts RGB to grayscale and uses the luminance as mask value.
+        /// </summary>
+        public enum MaskMode
```

Comment:
```suggestion
        [EditorBrowsable(EditorBrowsableState.Never)]
        public enum MaskMode
```

Please add browsable state never here.

Reply:
Done!

Reply:
Please avoid to define nested enum. It will be better to place outside of the class with other name for example `MaskEffectMode`